web/src/components/chat/main.tsx

```
@@ -49,12 +49,8 @@ const MainContent = () => {
         <>
           <WonkTop>
             <ChatHeader>
-              Meet PolicyWonk, a resource for navigating UC Davis administrative
-              policies and procedures.
-              <br />
-              <Link className='btn btn-wonk mt-2' href='/about'>
-                Learn more
-              </Link>
+              <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)